lib/seahorn/BvOpSem2.cc

```diff
@@ -1488,7 +1490,24 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemBase {
     }
 
     if (F.getName().equals("shadow.mem.global.init")) {
-      WARN << "Skipping initialization of a global: " << inst << "\n";
+      WARN << "initializing a global: " << inst << "\n";
+      Expr memOut = m_ctx.mkRegister(inst);
+      Expr memIn = m_ctx.getRegister(*CS.getArgument(1));
+      m_ctx.read(memIn);
+      setValue(inst, havoc(inst));
+
+      m_ctx.setMemReadRegister(memIn);
+      m_ctx.setMemWriteRegister(memOut);
+      m_ctx.setMemScalar(extractUniqueScalar(CS) != nullptr);
+
+      llvm::GlobalVariable *gv = (GlobalVariable *)((*CS.getArgument(2)).stripPointerCasts());
+      Expr x = op::array::select(m_ctx.read(memIn), lookup(*gv));
+      WARN << "x: ";
+      x->dump();
+      //m_ctx.MemCpy(lookup(*CS.getArgument(1)), lookup(*gv), 4, 4);
```

Comment:
This should be something like
```
auto gvVal = m_ctx.getGlobalVariableInitValue(gv)
if (gvVal.first) {
  m_ctx.MemCpy(lookup(*gv), gvVal.first, gvVal.second, 4);
}
```
Alignment is determined by the semantic machine since it decides how to allocate and align globals.
You may need to add API for `getGlobalVariableInitValue`. I've only added it to the memory manager, 
and it is probably not accessible directly from the context.